### ISSUE TITLE
Adopter - KGS Software GmbH

### ISSUE THREAD

**Issue by jsaalfeld** [kluctl/www-kluctl.io#63]
### What is the name of your company?

KGS Software GmbH

### What does your company do and how do you use kluctl?

KGS Software was founded in 2000 and creates an archive middleware with certified connections via SAP ArchiveLink, SAP ILM, SAP CMIS or for any other application via REST API. We rewrote the software to be cloud-centered with Kubernetes as a core technology stack.

KGS Software GmbH was in a hybrid position where some applications needed in the daily works were already in a k8s-environment while others were in a local, containerized context. Before integrating kluctl, the process of deploying applications was heavily dependent on manual work and maintaining test-environments was a labor-intense task. Setting up new environments or helping colleges entering the k8s-world was more about learning processes, rather than technology. With the ability of templating helm-values or k8s-yamls we were able to deploy new clusters faster and reduced the work in labor on the task and understanding the process immensely, while gaining a lot of technical insight from the diff-features.
Providing test-environments and getting used to the new context was made easier for the DevOps and more accessible and understandable for the  Software Engineers, Consultants and Supporters.

### URL to your company logo

https://jochen-saalfeld.de/logo-kgs-secondary-fullcolor-positive.png

### authorization

- [X] I agree to the statement above.

**Comment by codablock** [comment]
@jsaalfeld Here is a preview of the adopters page: https://deploy-preview-97--kluctl.netlify.app/

I've also updated the issue template to gather the required info in the future. As some of that was missing in the initial template, I tried to come up with my own stuff in the preview. Please inform me if you are fine with the current version or if you prefer to change something.

**Comment by jsaalfeld** [comment]
Thank you for coming up with something! Looks really good, but I'd like to change

"and reduced the work in labor on the task and understanding the process immensely."

to

"and massively reduces the workload within the process and substantially increases the understanding."

**Comment by codablock** [comment]
@jsaalfeld Is it ok for you to shorten the testimonial a bit? I just got feedback from someone who is much better at marketing stuff then me and his stance was that testimonials should be as short as possible as people are never fully reading them.

What about:
Since we integrated Kluctl, we were able to reduce the manual workloads massively. With templating of Helm-values and k8s-yamls, we can now easily maintain multiple envrironments with much less technical and mental overhead.
